platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -1749,6 +1749,12 @@ DisplayServer::WindowID DisplayServerX11::create_sub_window(WindowMode p_mode, V
 		rendering_device->screen_create(id);
 	}
 #endif
+
+	window_set_exclusive(id, p_exclusive);
```

Comment:
```suggestion
```
It's not implemented for X11 (I do not think there's such thing as exclusive child in X11), so the call is not necessary.